ui/src/components/Investor/OrderForm.tsx

```diff
@@ -84,6 +86,7 @@ const OrderForm: React.FC<Props> = ({
     const { ccpMap } = useRegistryLookup();
     const placeOrder = async (kind: OrderKind, deposits: DepositInfo[], amount: string) => {
         const key = wrapDamlTuple([exchange, operator, investor]);
+        const pair = {_1: tokenPair[0], _2: tokenPair[1]};
```

Comment:
It's unclear to me why `wrapDamlTuple` works with `daml-react` in other places, but not here. I'm guessing it has to do with the fact that it is a tuple of `Id`s, whereas elsewhere it is mostly of strings/parties, but this makes the typechecker happy.